cogs/annual_handover_and_reset.py

```diff
@@ -222,12 +220,13 @@ async def annual_roles_reset(self, ctx: TeXBotApplicationContext) -> None:
             for year_role_member in year_role.members:
                 await year_role_member.remove_roles(
                     year_role,
-                    reason=f"{ctx.user} used TeX-Bot slash-command: \"annual_roles_reset\"",
+                    reason=f'{ctx.user} used TeX-Bot slash-command: "annual_roles_reset"',
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes